### DISCUSSION TITLE
Encontrei um pequeno errinho ortográfico. Como sugerir a correção?

### DISCUSSION THREAD

**Discussion by mathiasbarreto** [campuscode/rails-guides-pt-BR#816] [Q&A] [answered]
Dá pra entender perfeito, mas falou um 's' ali no 'exitem' para ser 'existem'.
<img width="759" alt="image" src="https://user-images.githubusercontent.com/61096828/203666162-9fe25576-97ca-4724-822d-80cee0536fc9.png">

**Comment by HenriqueMorato** [comment, marked as answer]
Oi @mathiasbarreto você pode seguir o [Guia de contribuição](https://github.com/campuscode/rails-guides-pt-BR/blob/main/CONTRIBUTING.md) se quiser fazer a contribuição para a alteração 😄
